app/models/arm.rb

```diff
@@ -44,13 +44,18 @@ class Arm < ActiveRecord::Base
   after_save :update_liv_subject_counts
 
   validates :name, presence: true
-  validates :name, format: { with: /\A([A-Za-z0-9][A-Za-z0-9]*([ ][A-Za-z0-9])?)*\z/, 
-                   message: "can not contain any of the following characters: [ ] * / \\ ? : " }
+  validate :name_formatted_properly
   validate :name_unique_to_protocol
 
   validates :visit_count, numericality: { greater_than: 0 }
   validates :subject_count, numericality: { greater_than: 0 }
 
+  def name_formatted_properly
+    if !name.blank? && name.match(/\A([A-Za-z0-9`~!@#$%^&()\-_+={}|<>.,;'"]+(([ ][A-Za-z0-9`~!@#$%^&()\-_+={}|<>.,;'"]*)*)?)\z/).nil?
```

Comment:
I think this regex is equivalent, while being simpler: `\A([A-Za-z0-9``~!@#$%^&()\-_+={}|<>.,;'"][ ]*)+\z`